config/networks/other-l1s.json

```diff
@@ -277,48 +232,66 @@
         "https://aurora-testnet.drpc.org"
       ],
       "symbol": "ETH",
-      "tags": [
-        "deprecated"
-      ],
+      "tags": ["deprecated"],
       "type": "evm"
     },
     {
       "average_blocktime_ms": 5000,
       "chain_id": 122,
-      "explorer_urls": [
-        "https://explorer.fuse.io"
-      ],
-      "features": [
-        "eip1559"
-      ],
+      "explorer_urls": ["https://explorer.fuse.io"],
+      "features": ["eip1559"],
       "is_testnet": false,
       "network": "fuse",
       "required_confirmations": 6,
-      "rpc_urls": [
-        "https://rpc.fuse.io"
-      ],
+      "rpc_urls": ["https://rpc.fuse.io"],
       "symbol": "FUSE",
       "tags": [],
       "type": "evm"
     },
     {
       "average_blocktime_ms": 5000,
       "chain_id": 123,
-      "explorer_urls": [
-        "https://explorer.fusespark.io"
-      ],
-      "features": [
-        "eip1559"
-      ],
+      "explorer_urls": ["https://explorer.fusespark.io"],
+      "features": ["eip1559"],
       "is_testnet": true,
       "network": "fuse-spark",
       "required_confirmations": 6,
-      "rpc_urls": [
-        " https://rpc.fusespark.io"
-      ],
+      "rpc_urls": [" https://rpc.fusespark.io"],
       "symbol": "SPARK",
       "tags": [],
       "type": "evm"
+    },
+    {
+      "type": "evm",
+      "network": "peaq-mainnet",
+      "chain_id": 3338,
+      "explorer_urls": [
+        "https://peaq.subscan.io/",
+        "https://polkadot.js.org/apps/?rpc=wss://peaq.api.onfinality.io/public-ws#/explorer"
+      ],
+      "is_testnet": false,
+      "required_confirmations": 3,
+      "rpc_urls": [
+        "https://quicknode1.peaq.xyz",
+        "https://quicknode2.peaq.xyz",
+        "https://quicknode3.peaq.xyz"
+      ],
+      "average_blocktime_ms": 6000,
+      "features": ["eip1559"],
+      "symbol": "PEAQ"
+    },
+    {
+      "from": "peaq-mainnet",
+      "type": "evm",
+      "network": "peaq-agung",
+      "chain_id": 9990,
+      "explorer_urls": [
+        "https://agung-testnet.subscan.io/",
+        "https://polkadot.js.org/apps/?rpc=wss://wss-async.agung.peaq.network#/explorer"
```

Comment:
same here

Reply:
Do we need multiple block explorer Urls here, can we just use subscan endpoint?